pkgs/development/python-modules/django/2_2.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, buildPythonPackage, fetchPypi, substituteAll,
+  isPy3k,
+  geos, gdal, pytz, sqlparse,
+  withGdal ? false
+}:
+
+buildPythonPackage rec {
+  pname = "Django";
+  version = "2.2";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "06d1x1a4vqzwmv73sixpcf993fj2rxzrl61617hi8387zgj46dbw";
+  };
+
+  patches = stdenv.lib.optionals withGdal [
+    (substituteAll {
+      src = ./1.10-gis-libs.template.patch;
+      geos = geos;
+      gdal = gdal;
+      extension = stdenv.hostPlatform.extensions.sharedLibrary;
+    })
+  ];
+
+  # patch only $out/bin to avoid problems with starter templates (see #3134)
+  postFixup = ''
+    wrapPythonProgramsIn $out/bin "$out $pythonPath"
+  '';
```

Comment:
I don't think this does what the comment says it does. Instead, it wraps a second time creating e.g. `..django-admin-wrapped-wrapped`.